File: ios/PageCall/Model/Room.swift
import Foundation

struct Room: Codable {
    var id: String
    var name: String
    var pcaRoomId: String
    var start: String?
    var end: String?
    var participant: Int?
    var createdAt: String?
    var updatedAt: String?
}

struct Rooms: Codable {
    var liveRooms: [Room]
    var closedRooms: [Room]
}

struct Constants {
    static let endpoint = "http://3.38.3.80:8080/"
}

struct User {
    var id: String
    var name: String
    var nickName: String
}

File: web/src/models/room.ts
export type LiveRoom = {
  id: string;
  name: string;
  pcaRoomId: string;
  start: string;
  end: null;
  participant: number;
  createdAt: string;
  updatedAt: string;
};

export type ClosedRoom = {
  id: string;
  name: string;
  pcaRoomId: string;
  start: string;
  end: string;
  participant: number;
  createdAt: string;
  updatedAt: string;
};
